App.js

```diff
@@ -1,21 +1,16 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, ActivityIndicator} from 'react-native';
+import Results from './components/Results';
 
 export default class App extends React.Component {
-  render() {
-    return (
-      <View style={styles.container}>
-        <Text>Open up App.js to start working on your app!</Text>
-      </View>
-    );
-  }
-}
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+  // let url = 'https://world.openfoodfacts.org/api/v0/product/' + barcode + '.json'
+
+ render() {
+   return(
+     <Results />
+   )
+
+ }
+
```

Comment:
Blank spaces may be worth removing where we're no longer editing to clean up our code... don't want this becoming a code-pocalypse.

Reply:
@Din-s1 Removing that now, thanks!